setup.cfg

```diff
@@ -91,6 +91,7 @@ ignore_missing_imports = True
 strict = True
 
 [coverage:run]
+source = build, tests
```

Comment:
Just a note that this does not work for subprocess calls that change the cwd... https://twitter.com/gjbernat/status/1452419280394461195 and we might have some of those around...

Reply:
Good to know, though looking at the Codecov report, I don't think we're affected by it.